### ISSUE TITLE
508-defect-2 [SEMANTIC MARKUP]: Tab panel should not be nested within a nav

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#24582]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screen reader user, I expect to see a list of links when I reach a navigation landmark, not a tab panel.

## Details

The `nav` that wraps the tab panel may confuse screen reader users on its role and use. It also lacks an accessible label which will cause two generic "navigation" landmarks to appear in a screen reader's element list or rotor (see video).

https://user-images.githubusercontent.com/14154792/117890587-71860300-b283-11eb-9687-c9a92d393eee.mov

## Acceptance Criteria

- [ ] Tab panel is no longer wrapped in a `nav`

## Proposed Solution

Remove the `nav` that wraps the tab panel

## WCAG or Vendor Guidance (optional)

* [WAI-ARIA: Tabs with Automatic Activation](https://www.w3.org/TR/wai-aria-practices-1.2/examples/tabs/tabs-1/tabs.html)

**Comment by mdewey** [comment]
Hey @joshkimux! This has been addressed in staging.

**Comment by joshkimux** [comment]
Confirmed this has been fixed 🥳 
<img width="1555" alt="Screen Shot 2021-05-24 at 8 36 42 AM" src="https://user-images.githubusercontent.com/14154792/119348692-387a6500-bc6b-11eb-904d-c1d0ec5b5dda.png">
